bentoml/_internal/bento/README.md

```diff
@@ -1,11 +1,157 @@
+infer all PyPI packages that are required by the service code, and pin down to the version
+used in current environment.
+
+User may further customize the build of a service as below:
+
+```python
+import bentoml
+
+bentoml.build(
+    service="fraud_detector.py:svc",
+    version="any_version_label",  # override default version generator
+    description=open("README.md").read(),
+    include=['*'],
+    exclude=[], # files to exclude can also be specified with a .bentoignore file
+    additional_models=["iris_model:latest"], # models to pack in Bento, in addition to the models required by service's runners 
+    labels={
+        "foo": "bar",
+        "team": "abc"
+    },
+    python=dict(
+        packages=["tensorflow", "numpy"],
+        # requirements_txt="./requirements.txt",
+        index_url="http://<api token>:@mycompany.com/pypi/simple",
+        trusted_host=["mycompany.com"],
+        find_links=['thirdparty..'],
+        extra_index_url=["..."],
+        pip_args="ANY ADDITIONAL PIP INSTALL ARGS",
+        wheels=["./wheels/*"],
+    ),
+    docker=dict(
+        # "base_image": "mycompany.com/registry/name/tag",
+        distro="amazonlinux2",
+        gpu=True,
+        setup_script="setup_docker_container.sh",
+        python_version="3.8",
+    ),
+    conda={
+        "environment_yml": "./environment.yml",
+        "channels": [],
+        "dependencies": [],
+    }
+)
+```
+
+
+Alternatively, user may put all build options in a `bentofile.yaml` in the same directory
+```yaml
+service: "iris_classifier:svc"
+description: "./readme.md"
+labels:
+  foo: bar
+  team: abc
+include:
+- "*.py"
+- "*.json"
+exclude: 
+- "*.pyc"
+additional_models:
+- "iris_model:latest"
+docker:
+  distro: slim
+  gpu: True
+  python_version: "3.8"
+  setup_script: "./setup_env.sh"
+python:
+  packages:
+    - tensorflow
+    - numpy
+    - --index-url http://my.package.repo/simple/ SomePackage
+    - --extra-index-url http://my.package.repo/simple SomePackage
+    - -e ./my_py_lib
+  index_url: http://<api token>:@mycompany.com/pypi/simple
+  trusted_host: mycompany.com
+  # index_url: null # means --no-index
+  find_links:
+    - file:///local/dir
+    - thirdparth...
+  extra_index_urls:
+    - abc.com
+  pip_args: "-- "
+  wheels:
+    - ./build/my_lib.whl
+```
+
+Another bentofile example:
+```yaml
+service: "foo.bar.another_svc:my_svc"
+include:
+- "another_svc/**"
+docker:
+  base_image: "my_own_docker_image:0.1.2"
+python:
+  requirements_txt: "./requirements.txt"
+  index_url: "http://<api token>:@mycompany.com/pypi/simple"
+conda:
+  channels: ["h2o"]
+  dependencies: ["h2o"]
+```
+
+
+When running the CLI command `bentoml build` without any parameter, it will look for
+a `bentofile.yaml` file in current directory and use it as build target. User may also
+specify which build file to use with the `-f` option, e.g.:
+
+```bash
+bentoml build -f ./bentofile-proj2.yaml
+```
```

Comment:
Is there a use case for having multiple build files?

Reply:
yes, when there are multiple bento service defined in the same python project